contrib/localnet/bitcoin-sidecar/js/src/client.ts

```
@@ -0,0 +1,183 @@
+import { ScriptBuilder } from "./script";
+import { Taptree } from "bitcoinjs-lib/src/types";
+import { toXOnly } from "./util";
+
+const LEAF_VERSION_TAPSCRIPT = 0xc0;
+
+initEccLib(ecc);
+const bip32 = BIP32Factory(ecc);
+const rng = randomBytes;
+
+/// The evm address type, a 20 bytes hex string
+export type Address = String;
```

Biome

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)